src/proptests.rs

```diff
@@ -94,6 +97,22 @@ impl PropTestEnv {
         range.end += 1;
     }
 
+    pub async fn multi_append(&mut self, queue: &str, count: u64) {
+        let range = self.state.get_mut(queue).unwrap();
+
+        let res = self
+            .record_log
+            .append_records(queue, Some(range.end), (0..count).map(|_| &[] as &[u8]))
```

Comment:
low importance nitpick: I tend to prefer
 
`std::iter::repeat(&b""[..]).take(count)`